doc.md

```diff
@@ -0,0 +1,110 @@
+# Hackseq 2017
+
+The system allows users to run graphical user interface application (e.g. `igv`), provides 100 GB hard memory and 20 GB soft memory, maps host users to Docker users inside containers and mounts team folders. 
+
+## Overview
+
+**Servers**: orca01.dmz.bcgsc.ca, orca02.dmz.bcgsc.ca
+* 5 teams per a server
+* a team shares a team’s folder
+* the Docker container per a team
+* each user has a home directory
+* ability to run GUI applications (X11)
+* the groups’ user name is the same as a name of the team
+
+## Group creation
+
+Add groups: `orca_users:*:1000:`, name of teams (e.g. `hackseq17_1:*:1002:`)
+
+## User creation
+* Primary group name = a name of the team = a name of the Docker container
+* Secondary group name = `1000(orca_users)`
+* User home directory: 
+    1) create empty `.Xauthority file` (X11)  
+    2) to make `brew install` work:
```

Comment:
Elaborate.
`brew` requires write permissions to the user's `~/.cache/Homebrew` directory.